plugin.yml

```diff
@@ -0,0 +1,31 @@
+name: setup-oblt-cli
+description: Grant pipelines access to Google Cloud resources using Workload Identity Federation
+author: https://github.com/elastic
+public: true
+requirements:
+  - buildkite-agent
```

Comment:
what's the reason for this requirement?

Reply:
Oh, I will remove it again, for now.

I was thinking, maybe we want to set the cluster id as meta-data or something.

But this requires a create command or any sort of command, which might be added in the future.

Reply:
https://github.com/elastic/oblt-cli-buildkite-plugin/pull/1/commits/44d62d691fe7c91f23c2b3b3381ba60873fd14cf